### PR TITLE
C/ulimit memlock docs

### PR DESCRIPTION
**Pull request by celaus**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,14 +65,14 @@ To form a cluster from scratch, start a few instances of the Crate container as
...
+# docker run -d crate/crate crate
 ```
 
 To access the admin UI, map port 4200 and point your browser to port tcp/4200 of
 a node of your choice while you start it or look up its IP later:
 
 ```console
-# firefox "http://$(docker inspect --format='{{.NetworkSettings.IPAddress}}' $(docker run -d crate/crate)):4200/admin"
+# firefox "http://$(docker inspect --format='{{.NetworkSettings.IPAddress}}' $(docker run -d crate/crate crate)):4200/admin"
 ```
```
Comment (Nils-Magnus) [inline README.md]: Not necessary to give the name of the program to start when it's `crate` and doesn't come with extra args.

File: README.md [hunk 2]
```diff
@@ -78,7 +81,7 @@ the ports from the Docker container to the host it acts like a native
 installation. Crate's default ports 4200 (HTTP) and 4300 (Transport protocol).
 
 ```console
-# docker run -d -p 4200:4200 -p 4300:4300 crate/crate
+# docker run -d -p 4200:4200 -p 4300:4300 crate/crate crate
```
Comment (Nils-Magnus) [inline README.md]: 3rd time :)

File: README.md [hunk 3]
```diff
@@ -87,7 +90,7 @@ Crate stores all important data in _/data_. It's advised to mount this
 directory to avoid writing within the docker image:
 
 ```console
-# docker run -d -v <data-dir>:/data crate/crate
+# docker run -d -v <data-dir>:/data crate/crate crate
```
Comment (Nils-Magnus) [inline README.md]: 4th

**Comment by Nils-Magnus** [comment]
Ack

**Comment by celaus** [comment]
I think we should leave them for consistency

**Pull request merged**